rllib/evaluation/episode_v2.py

```diff
@@ -295,7 +296,7 @@ def postprocess_episode(
 
             if (
                 not pre_batch.is_single_trajectory()
-                or len(set(pre_batch[SampleBatch.EPS_ID])) > 1
+                or len(np.unique(pre_batch[SampleBatch.EPS_ID])) > 1
```

Comment:
Because of the above changes, EPS_IDs turn out to be np arrays as well, so set does not work here anymore.